Helper/Api/Orders.php

```diff
@@ -618,4 +598,143 @@ public function getOrders($storeId, $page, $pageSize): OrderSearchResultInterfac
+        }
+
+        return $itemPrice;
+    }
+
+    /**
+     * @param $item
+     * @param $taxFreeId
+     * @return array
+     */
+    private function getBundlePayload($item, $taxFreeId)
+    {
+        $bundleItemsPriceTotal = null;
+        $bundlePrice = $item->getPriceInclTax();
+
+        $parentProduct = $this->getParentProductData($item);
+
+        foreach ($item->getChildrenItems() as $bundleItem) {
+            $bundleItemSku = $bundleItem->getSku();
+            $bundleItemPrice = $bundleItem->getProduct()->getPrice();
+            $bundleItemsPriceTotal += $bundleItemPrice;
+            $bundleItemData = $this->jsonSerializer->unserialize(
+                $bundleItem->getProductOptions()['bundle_selection_attributes']
+            );
+            $bundlePayload[] = [
+                'quantity' => $bundleItem->getQtyOrdered(),
+                'ppu_wt' => $bundleItemData['price'],
+                'sku' => $bundleItemSku,
+                'name' => $bundleItem->getName(),
+                'description' => $bundleItemData['option_label'],
+                'tax_rate_id' => $this->getTaxRateId($item, $taxFreeId),
+                'extra' => [
+                    'external_id' => $bundleItem->getProduct()->getId(),
+                    'options' => [
+                        'option' => $bundleItemData['option_label']
+                    ],
+                    'parent_product' => $parentProduct
+                ]
+            ];
+        }
+
+        $bundleDiscount = $this->getBundleDiscount((float)$bundleItemsPriceTotal, (float)$bundlePrice);
+
+        if ($bundleDiscount) {
+            $bundlePayload[] = $this->getBundleDiscountData($bundleDiscount, $parentProduct);
+        }
+
+        return $bundlePayload;
+    }
+
+    private function getGeneralPayload($item, $taxFreeId)
+    {
+        $shopProductId = '';
+
+        if ($item->getProduct() !== null && $item->getProduct()->getStorekeeperProductId()) {
+            $shopProductId = $item->getProduct()->getStorekeeperProductId();
+        }
+
+        $payloadItem = [
+            'sku' => $item->getSku(),
+            'quantity' => $item->getQtyOrdered(),
+            'name' => $item->getName(),
+            'shop_product_id' => $shopProductId,
+        ];
+
+        if (((float)$item->getTaxAmount()) > 0) {
+            $payloadItem['ppu_wt'] = $item->getPriceInclTax();
+            $payloadItem['before_discount_ppu_wt'] = (float) $item->getOriginalPrice();
+        } else {
+            $payloadItem['ppu_wt'] = $this->getItemPrice($item);
+        }
+
+        $payloadItem['tax_rate_id'] = $this->getTaxRateId($item, $taxFreeId);
+
+        return $payloadItem;
+    }
+
+    private function getParentProductData($item)
+    {
+        return [
+            'external_id' => $item->getId(),
+            'sku' => $item->getSku(),
+            'name' => $item->getName()
+        ];
+    }
+
+    private function getBundleDiscount($bundleItemsPriceTotal, $bundlePrice)
```

Comment:
```suggestion
    private function getBundleDiscount($bundleItemsPriceTotal, $bundlePrice)
```
please add type hinting to all of the functions in that file, also for returns